pyrefinebio/config.py

```diff
@@ -9,12 +8,32 @@ class Config:
 
     _instance = None
 
-    def __new__(cls, config_file=None):
+    def __new__(cls):
         if not cls._instance:
             cls._instance = super(Config, cls).__new__(cls)
 
             cls.config_file = (
-                os.getenv("CONFIG_FILE") or config_file or str(Path.home()) + "/.refinebio.yaml"
+                os.getenv("CONFIG_FILE") or str(Path.home()) + "/.refinebio.yaml"
             )
 
+            if not os.path.exists(cls.config_file):
+                with open(cls.config_file, 'w'): pass
```

Comment:
I think we should wait until the user asks us to save something to the config to create the file. I'd probably be a little annoyed if I used a command line tool for something trivial and it littered an empty file in my home dir unnecessarily.